### ISSUE TITLE
Speedbump on Clear button

### ISSUE THREAD

**Issue by AndrewMohawk** [redcanaryco/mac-monitor#10]
**At a high level -- can you summarize your request?**
I occasionally accidentally hit `clear` and lose all my captured data. It would be nice if this had a "Are you sure?" dialog

**What is the current alternative solution?**
Avoid clicking Clear!

**Are there "In-the-Wild" threats or corresponding ATT&CK techniques that exist for which this telemetry would be helpful?**
N/A

**Comment by Brandon7CC** [comment]
GREAT idea! 😆 I've done the same

**Comment by Brandon7CC** [comment]
@AndrewMohawk same w/"⌘+q"? Similar to Google Chrome we can ask to confirm before quit.

**Comment by AndrewMohawk** [comment]
Yeah, any real speedbump to stop me from accidentally clearing my data since I seem to regularly hit the buttons on either side! (even just moving clear would work for me!). Perhaps the button could also be a little more descriptive, when I first used it I assumed it cleared my filters!

**Comment by Brandon7CC** [comment]
Implemented in `v1.0.4` thank you for the excellent suggestion Andrew! More good stuff to come :)
